### PR TITLE
metrics: Use sudo with ctr in common.bash

### LINKED ISSUE #5495: update metrics lib common.sh to use sudo with ctr

### PR DESCRIPTION
**Pull request by GabyCT**
This PR uses sudo with ctr in common.bash as part of consistency as in other functions that are using ctr or docker.

Fixes #5495

### CODE REVIEW

File: metrics/lib/common.bash [hunk 1]
```diff
@@ -70,7 +70,7 @@ check_images()
 	local img req_images=( "$@" )
 	for img in "${req_images[@]}"; do
 		echo "ctr pull'ing: $img"
-		if ! ctr image pull "$img"; then
+		if ! sudo ctr image pull "$img"; then
```
Comment (dborquez) [inline metrics/lib/common.bash:73]: For uniformity, could you use $CTR_EXE instead of the command itself?

**Review by dborquez — CHANGES_REQUESTED** [review]

File: metrics/lib/common.bash (hunk 1 above)
Comment (GabyCT) [reply metrics/lib/common.bash]: @dborquez changes applied

**Review by dborquez — APPROVED** [review]
thank you @GabyCT

**Review by cmaf — APPROVED** [review]

**Pull request merged**